Repeat/repeat.py

```diff
@@ -0,0 +1,56 @@
+
+from PIL import Image
+import numpy as np     # numpy
+from collections import namedtuple
+
+repeat = namedtuple('repeat', 'count pixel')
```

Comment:
Made the change. But this was how most of the namedtuple examples online were using it.